### DISCUSSION TITLE
Configuration option to start codespaces in a new tab

### DISCUSSION THREAD

**Discussion by ntindle** [community/community#8163] [Codespaces] [answered]
When I click `New codespace` it opens in the same tab. I would prefer to keep the repo open and codespaces open in a new tab as I often need to reference the repo for various pieces of information. 

I understand that having it open in the same tab seems convenient but I almost always have the repo open for reference while I work.

**Comment by asciimike** [comment, marked as answer]
@ntindle thanks for the feedback! We'll be switching it to always open in a new tab, as we have found that most people prefer the desired behavior. Stay tuned to [github.blog/changelog](https://github.blog/changelog) for the latest.
